src/cmd.fs

```diff
@@ -87,16 +85,18 @@ module Cmd =
     let ofSub (sub: Sub<'msg>) : Cmd<'msg> =
         [sub]
 
+#if FABLE
```

Comment:
Should be `#if FABLE_COMPILER` (as well as for other instances)

Reply:
Ah thanks a lot!